src/components/common/disclosure.js

```diff
@@ -1,21 +0,0 @@
-import React from 'react'
-import {
-  Disclosure,
-  DisclosureButton,
-  DisclosurePanel,
-} from '@reach/disclosure'
```

Comment:
Since we're removing the only use of this package, let's remove it from `package.json` as well.

Reply:
fixed in e2dbf13a0aa2b7b70c233d2fa8f6d0cf8331676e